magento/pwa-studio
# Cypress Setup

## Description

Initial cypress setup.

## Related Issue
Closes PWA-1569

### Verification Stakeholders
@dpatil-magento 

### Specification



### Verification Steps
1. Checkout the branch. Navigate to `pwa-automation-tests`.
2. Run `yarn install` if its your first time.
3. Run `yarn test` to open up the cypress UI. You should see 2 dummy tests but they won't run. They are just provided as examples.

## Screenshots / Screen Captures (if appropriate)

![image](https://user-images.githubusercontent.com/35203638/112367424-e2f3fd00-8ca7-11eb-9d1a-7e671450762e.png)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
